src/dm/dm_radio.cpp

```diff
@@ -219,7 +219,7 @@ bool dm_radio_t::operator == (const dm_radio_t& obj) {
 
 void dm_radio_t::operator = (const dm_radio_t& obj)
 {
-	strncpy(this->m_radio_info.id.net_id, obj.m_radio_info.id.net_id, strlen(obj.m_radio_info.id.net_id) + 1);
+	memmove(this->m_radio_info.id.net_id, obj.m_radio_info.id.net_id, strlen(obj.m_radio_info.id.net_id) + 1);
```

Comment:
Same as with the previous `memmove`:
```suggestion
	memcpy(this->m_radio_info.id.net_id, obj.m_radio_info.id.net_id, sizeof(em_long_string_t));
```